bundles/AdminBundle/Controller/Admin/Document/PageController.php

```diff
@@ -254,23 +254,23 @@ public function checkPrettyUrlAction(Request $request)
             ]);
         }
 
-        $message = '';
```

Comment:
why not refactor to use symfony validation...

Reply:
I did not know that it was possible to validate non-Symfony forms until you asked.

But I think I'm going to let this stand as is because the time/effort it is going to take to do such a refactor.

Good idea for 7.0 though :)

Reply:
you can validate basically everything, it is independent from symfony form.

As it is not a BC break, it would work perfectly for 6.* and the time effort wouldn't be too bad.... But, if you don't want to spent the time, I totally understand that.